notebooks/quickstart/data-scientist/02-explore-dataset.ipynb

```diff
@@ -0,0 +1,481 @@
+{
```

Comment:
<p><strong>Step 10: Publish the results.</strong></p><p> </p><p>Talk about publish here.</p><p>First cell</p><pre class="ql-syntax" spellcheck="false">mean_age_public =ages.publish(sigma=20)
</pre><p> </p><p>Second cell: Check if results to the pointer are ready on the server side. (Status: Ready indicates that results are ready on the server side.)</p><pre class="ql-syntax" spellcheck="false">mean_age_public
</pre><p> </p><p><strong>Step 11: Download the published results</strong></p><p>Talk about .get here.</p><pre class="ql-syntax" spellcheck="false">mean_age_public.get(delete_obj=False)
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/7045/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>